glommio/src/sync/rwlock.rs

```diff
@@ -917,7 +917,7 @@ impl<T: Default> Default for RwLock<T> {
 
 impl<T> Drop for RwLock<T> {
     fn drop(&mut self) {
-        self.close();
+        self.close().unwrap();
```

Comment:
Isn't this problematic?

Reply:
No, Guard can not outlive the lock itself (lifetime annotation prohibits that), so at this moment all locks are bound to be freed. I will add a comment to this line.